src/main/webapp/stylesheets/instructorFeedbackResults.css

```diff
@@ -0,0 +1,93 @@
+/* borrowed from Bootstrap's .panel-default properties */
```

Comment:
Alright, then I'll make it to panel-default (and btn-default, while we're at it). Allow me some time to rewrite all the affected test pages as well.